### PR TITLE
Updating README

### PR DESCRIPTION
**Pull request by AnHeuermann**
## Related Issue

#13, #14

### CODE REVIEW

**Comment by AnHeuermann** [comment]
We need to sort out the license before merging this.

File: README.md [hunk 1]
```diff
@@ -1,12 +1,139 @@
...
+- [OpenModelica](https://openmodelica.org)
+- [Python](https://www.python.org/)
+  - datetime.py
+  - matplotlib.py
+  - monotonic.py
+  - natsort.py
+  - joblib.py
+  - ompython.py
+  - simplejson.py
+  - psutil.py
```
Comment (sjoelund) [inline README.md:19]: I wouldn't list them. They are not called .py anyway

File: README.md [hunk 2]
```diff
@@ -1,12 +1,139 @@
...
+- Upload and backup
+  - Upload HTML files somewhere
+  - backup sqlite3.db
+
+### Reference Results
+
+For many available Modelica libraries result files are provide.
+If a library is tested that uses `MSLREFERENCE` or `REFERENCEFILES`
+inside `config/conf.json` and `config/conf-c++.json` one need to
+download the reference files to verify the simulation results.
```
Comment (sjoelund) [inline README.md:97]: But mostly you want to create your own json config and not use those

Comment (AnHeuermann) [reply README.md:97]: Most likely none besides us will ever use this.

The use case would be to check what the influence of a change in OpenModelica would be for all the libraries without needing to merge it into the master branch. E.g. with the new backend.

**Comment by AnHeuermann** [comment]
Should we merge this?
I don't have the rights to do so. Not sure about the license, though.

**Comment by casella** [comment]
I think we can merge it so we are making some progress, otherwise this PR risks getting stale. It is always possible to make other ones in the future. @sjoelund are you fine with that?

Regarding the license, I would use OSMC-PL as for the OMC source code. Which means, GPL v3 if you're not a Consortium Member, can do modifications and redistribution if you are a Level 2 member. I see no reason to do differently.

@sjoelund, @adrpo what do you think?

**Comment by sjoelund** [comment]
Some things, like OMPython, are using the OSMC-PL runtime license (BSD3)

**Comment by casella** [comment]
The runtime, of course we want people to be able to run OMC-generated code without any license issues, otherwise we'd become almost like any other commercial vendor. I'm not 100% sure why this would apply to OMPython's _source_ code.

In any case, this is a separate tool for testing of libraries, that we developed, and can probably be further improved. I guess the regular OSMC-PL suits better, because it forces people to either share the improvements they may make to it, or at least to join the OSMC if they don't want to.

**Comment by sjoelund** [comment]
GPLv3 doesn't require them to publish changes to the source code unless they publish the changed software. And given that it's Python code, if they do you pretty much get the changes.

**Comment by AnHeuermann** [comment]
Okay, I have added OSMC-PL to this PR.

**Pull request merged**